app/src/main/java/org/p2p/wallet/striga/ui/TopUpWalletPresenter.kt

```diff
@@ -29,19 +28,10 @@ class TopUpWalletPresenter(
     TopUpWalletContract.Presenter {
 
     private val isUserAuthByWeb3: Boolean
-        get() {
-            return seedPhraseProvider.getUserSeedPhrase().provider == SeedPhraseSource.WEB_AUTH ||
-                appFeatureFlags.strigaSimulateWeb3Flag.featureValue
-        }
+        get() = seedPhraseProvider.isWeb3AuthUser || appFeatureFlags.strigaSimulateWeb3Flag.featureValue
 
     private val strigaBankTransferProgress = MutableStateFlow(false)
 
-    init {
-        launch {
-            loadMetadataIfNot()
```

Comment:
no need to load metadata on open topup 
it's needed only on Bank transfer, so if it's available - try to load when bank transfer is clicked. 
we can explicitly see loading and errors in this case without random "Something went wrong" when top up is opened